service-front/module/Application/src/Forms/Country.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Forms;
+
+use Laminas\Form\Annotation;
+use Laminas\Hydrator\ObjectPropertyHydrator;
+use Laminas\Validator\NotEmpty;
+
+/**
+ * @psalm-suppress MissingConstructor
+ */
+#[Annotation\Hydrator(ObjectPropertyHydrator::class)]
+class Country
+{
+    /**
+     * @psalm-suppress PossiblyUnusedProperty
+     */
+    #[Annotation\Validator(NotEmpty::class, ['message' => 'Please select a country'])]
+    public mixed $country;
+
+    /**
+     * @psalm-suppress PossiblyUnusedProperty
+     */
+    #[Annotation\Validator(NotEmpty::class, ['message' => 'Please select a document'])]
+    public mixed $id_method;
```

Comment:
It would be good to validate these against a list of accepted values, to avoid saving something that we can't then work with